manifests/t/Thinkscape/ZeeDrive/65.13.0.0/Thinkscape.ZeeDrive.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Thinkscape.ZeeDrive
+PackageVersion: 65.13.0.0
+InstallerLocale: en-US
+InstallerType: msi
+Scope: user
```

```suggestion
Scope: machine
```

This should be changed to keep manifest consistency.